### PR TITLE
Create 12.pmw.hwc.csv

### PR DESCRIPTION
**Pull request by mucheinz**
Zeitfenster Zirkulationspumpe - Deutsch

### CODE REVIEW

File: ebusd-2.x.x/de/vaillant/12.pmw.hwc.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,,,
```
Comment (john30) [inline ebusd-2.x.x/de/vaillant/12.pmw.hwc.csv:1]: why not simply include timer.inc ?

Comment (mucheinz) [reply ebusd-2.x.x/de/vaillant/12.pmw.hwc.csv:1]: excellent idea - there is no *.inc file in the ebusd-2-x.x path as of now, just in the ebusd-2-1.x path. Does it need to be created here as well?

Comment (john30) [reply ebusd-2.x.x/de/vaillant/12.pmw.hwc.csv:1]: you should stick to the 2.1.x path anyway, the 2.x.x path is older and only kept for reference. no changes are going to be accepted in 2.x.x
this seems to be a duplicate of #177 anyway, right?

Comment (mucheinz) [reply ebusd-2.x.x/de/vaillant/12.pmw.hwc.csv:1]: #177 and #178 were for German and English respectively, but no longer needed here.

**Review by mucheinz — COMMENTED** [review]
no longer needed here

**Pull request closed without merging**